### PR TITLE
Added new routes to prescriptions.

### LINKED ISSUE #6939: Route of medicine- Add inhalation/ryles tube
**Is your feature request related to a problem? Please describe.**
Route in the prescription module is limited to a few options

**Describe the solution you'd like**
Add the following options:

- [ ] Inhalation
- [ ] Nasogastric/Gastrostomy tube
- [ ] intrathecal injection
- [ ] Transdermal 
- [ ] Rectal

### PR DESCRIPTION
**Pull request by sriharsh05**
## Proposed Changes
Added new routes to prescriptions.

 1.Inhalation
 2.Nasogastric/Gastrostomy tube
 3.intrathecal injection
 4.Transdermal
 5.Rectal

- Fixes #6939 
- Backend PR : https://github.com/coronasafe/care/pull/1808

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Medicine/CreatePrescriptionForm.tsx [hunk 1]
```diff
@@ -137,7 +137,17 @@ export default function CreatePrescriptionForm(props: {
...
+  "ORAL",
+  "IV",
+  "IM",
+  "SC",
+  "INHALATION",
+  "NASOGASTRIC",
+  "INTRATHECAL",
+  "TRANSDERMAL",
+  "RECTAL",
+];
```
Comment (rithviknishad) [inline src/Components/Medicine/CreatePrescriptionForm.tsx]:
```suggestion
] as const;
```

File: src/Components/Medicine/models.ts [hunk 2]
```diff
@@ -16,7 +16,16 @@ interface BasePrescription {
...
+  route?:
+    | "ORAL"
+    | "IV"
+    | "IM"
+    | "SC"
+    | "INHALATION"
+    | "NASOGASTRIC"
+    | "INTRATHECAL"
+    | "TRANSDERMAL"
+    | "RECTAL";
```
Comment (rithviknishad) [inline src/Components/Medicine/models.ts]:
```suggestion
  route?: (typeof PRESCRIPTION_ROUTES)[number];
```

**Review by rithviknishad — COMMENTED** [review]
LGTM; just a minor suggestion, there's a better way to manage types for these

**Comment by rithviknishad** [comment]
@sriharsh05 kindly link the backend PR too in the main comment once ready for testing

**Review by rithviknishad — APPROVED** [review]

**Review by Ashesh3 — APPROVED** [review]

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**